src/messages.c.jinja

```diff
@@ -0,0 +1,47 @@
+// NOTE: This file has been generated automatically by “{{script}}”.
+//       Do not edit manually!
+
+#include <stdlib.h>
+#include "messages.h"
+
+static const struct xkb_message_entry xkb_messages[] = {
+    {% for entry in entries %}
+    { {#--#}{{ entry.message_code_constant}}{#, "{{entry.id}}", "{{entry.description}}"#}},
+    {% endfor %}
+    {0{#, NULL, NULL #}}
+};
+
+const struct xkb_message_entry*
+xkb_message_get(xkb_message_code_t code)
+{
+{# Binary search seems overkill for now
+    if (code < XKB_LOG_MESSAGE_MIN_CODE || code > XKB_LOG_MESSAGE_MAX_CODE)
```

Comment:
Probably better to just remove this commented out code (can leave the "Binary search seems overkill for now" comment though).

Reply:
Done